lib/db/schema/patch-039-040.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE IF NOT EXISTS emailBounces (
+  email VARCHAR(255) NOT NULL,
```

Comment:
I feel like we had this conversation in a 1:1 before, and punted. I know I've certainly thought about it recently.